qiskit-superstaq/qiskit_superstaq/superstaq_job.py

```diff
@@ -85,7 +83,8 @@ def result(self, timeout: Optional[float] = None, wait: float = 5) -> qiskit.res
                 counts = dict((key[::-1], value) for (key, value) in counts.items())
             results_list.append(
                 {
-                    "success": True,
+                    "success": self._overall_status == "Done",
+                    "status": self._overall_status,
```

Comment:
think these ones should still be
```suggestion
                    "success": result["status"] == "Done",
                    "status": result["status"],
```
(that way individual sub-jobs can have their own statuses when the overall status is "Failed")

Reply:
Thanks for the catch, corrected in dcf6cca5a4f356abe8ec0610e19e16187cffa570